assets/js/components/KeyMetrics/widget-tiles/AdsenseTopEarningContentWidget.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
@jimmymadon @hussain-t, now I wonder whether the `components/KeyMetrics` folder is the right place for all these widgets... 🤔 All of these widgets are intended for specific modules, then maybe we should place them in those modules and register there as well? What do you think guys?

Reply:
Thanks, @eugene-manuilov. That sounds like a good idea. However, I would leave it to @jimmymadon since he is the owner of this epic 😃

Reply:
Thanks for the clarifications, @jimmymadon 👍 I will make the changes

Reply:
I agree with @eugene-manuilov that modules should register their own widgets ("normal" or key metrics) as they do now unless there is a reason we can't do that – although I can't think of one. @tofumatt reviewed the AC so I'm not sure if he has anything to add here. This also implies that the widget components themselves should also live in their respective modules as well so this sounds more like something we should do in a small follow-up issue to avoid last minute changes here.

Reply:
Thanks for your thoughts, @aaemnnosttv. I will create a follow-up ticket. Could you or @eugene-manuilov merge this PR?

Reply:
I have created a follow-up [ticket](https://github.com/google/site-kit-wp/issues/6510) and added AC and IB since it was pretty straightforward.